cobalt/js_profiler/BUILD.gn

```diff
@@ -0,0 +1,53 @@
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+static_library("js_profiler") {
+  sources = [
+    "profiler.cc",
+    "profiler.h",
+    "profiler_trace_builder.cc",
+    "profiler_trace_builder.h",
+    "profiler_trace_wrapper.h",
+  ]
+
+  deps = [
+    "//cobalt/base",
+    "//cobalt/browser:generated_bindings",
+    "//cobalt/browser:generated_types",
+    "//cobalt/dom",
+    "//cobalt/script",
+    "//cobalt/script/v8c:engine",
+    "//cobalt/web",
+    "//third_party/chromium/media:media",
+    "//third_party/v8:cppgc",
+  ]
+}
+
+target(gtest_target_type, "profiler_test") {
```

Comment:
One last nit, would it be worthwhile to rename this target to `js_profiler_test` as well ? The Python file would need to be updated accordingly.

Reply:
Done.